pxr/usdImaging/usdImaging/instanceAdapter.cpp

```diff
@@ -2510,18 +2522,34 @@ UsdImagingInstanceAdapter::GetScenePrimPaths(
         // invisible instances).
         VtIntArray indices = _ComputeInstanceMap(instancerPrim, *instrData, 
             _GetTimeWithOffset(0.0));
-        std::vector<int> remappedIndices;
 
-        remappedIndices.reserve(instanceIndices.size());
-        for (size_t i = 0; i < instanceIndices.size(); i++)
-            remappedIndices.push_back(indices[instanceIndices[i]]);
+		SdfPathVector result;
 
-        SdfPathVector result;
-        result.reserve(instanceIndices.size());
-        _GetScenePrimPathsFn primPathsFn(this, remappedIndices, proto.path);
-        _RunForAllInstancesToDraw(instancerPrim, &primPathsFn);
-        for (size_t i = 0; i < remappedIndices.size(); i++)
-            result.push_back(primPathsFn.primPaths[remappedIndices[i]]);
+        if (!instanceIndices.empty()) {
+            int minIdx = INT_MAX;
```

Comment:
Could we use either INT_MAX or std::numeric_limits<int>::max() consistently instead of using both?

Reply:
changed to std::numeric_limits::max()